pkg/requestutil/context_test.go

```diff
@@ -53,5 +55,17 @@ func (s *testRequestContextSuite) TestRequestInfo(c *C) {
 	c.Assert(result.IP, Equals, "localhost")
 	c.Assert(result.URLParam, Equals, "{\"id\"=1}")
 	c.Assert(result.BodyParam, Equals, "{\"state\"=\"Up\"}")
-	c.Assert(result.TimeStamp, Equals, "2022")
+	c.Assert(result.StartTimeStamp, Equals, timeNow)
+}
+
+func (s *testRequestContextSuite) TestExcutionInfo(c *C) {
```

Comment:
The name of the case should also be changed.

Reply:
fix